### PR TITLE
Added CV_CAP_PROP_SAR properties to videoio module

### PR DESCRIPTION
**Pull request by pletessier**
Hi,

There exist some videos with a SAR (storage aspect ratio) != 1:1, see below the ffprobe info of one of my video file: 
Stream #0:0[0x8ad]: Video: mpeg2video (Main) ([2][0][0][0] / 0x0002), yuv420p, 720x576 [SAR 64:45 DAR 16:9], 25 fps, 25 tbr, 90k tbn, 50 tbc

FFMpeg or vlc can both play them with the good display aspect ratio (16/9), but not OpenCV, so when I load them, the videos are stretched, and I have to resize the grabbed frames.

Furthermore, there are not any functions available to know the SAR.

In this PR, I propose to add two new properties : CV_FFMPEG_CAP_PROP_SAR_NUM and CV_FFMPEG_CAP_PROP_SAR_DEN. These properties are the numerator and the denominator of the SAR.

Pierre

### CODE REVIEW

**Comment by alalek** [comment]
Looks good to me.
Could you please resolve merge conflict (something like `git pull --rebase upstream master`)

**Comment by alalek** [comment]
Try to use something like this:
```
# return to previous state
git checkout -B aspect-ratio 925e816176d19d47481f7d84e19d82df1025522a
# try to merge upstream changes, will receive message about conflict - it is OK
git pull --rebase git://github.com/Itseez/opencv master 
# edit file videoio_c.h and after that stash them
git add modules/videoio/include/opencv2/videoio/videoio_c.h
# move forward
git rebase --continue
# force push changes on GitHub
git push -f <your github remote> aspect-ratio
```

**Comment by pletessier** [comment]
Thanks for the commands !

**Comment by alalek** [comment]
You are welcome!
:+1:

File: modules/videoio/src/cap_ffmpeg_api.hpp [hunk 1]
```diff
@@ -23,7 +23,9 @@ enum
     CV_FFMPEG_CAP_PROP_FRAME_HEIGHT=4,
     CV_FFMPEG_CAP_PROP_FPS=5,
     CV_FFMPEG_CAP_PROP_FOURCC=6,
-    CV_FFMPEG_CAP_PROP_FRAME_COUNT=7
+    CV_FFMPEG_CAP_PROP_FRAME_COUNT=7,
+    CV_FFMPEG_CAP_PROP_SAR_NUM=39,
+    CV_FFMPEG_CAP_PROP_SAR_DEN=40
```
Comment (alalek) [inline modules/videoio/src/cap_ffmpeg_api.hpp]: Could you please update values here also

**Pull request merged**